rustler_codegen/src/context.rs

```diff
@@ -0,0 +1,170 @@
+use proc_macro2::{Span, TokenStream};
+use syn::{Data, Field, Ident, Lit, Meta, NestedMeta, Variant};
+
+use super::RustlerAttr;
+
+///
+/// A parsing context struct.
+///
+/// `Context` holds information usable for different codegen modules.
+///
+pub(crate) struct Context<'a> {
+    pub(crate) attrs: Vec<RustlerAttr>,
+    pub(crate) ident: &'a proc_macro2::Ident,
+    pub(crate) ident_with_lifetime: proc_macro2::TokenStream,
+    pub(crate) variants: Option<Vec<&'a Variant>>,
+    pub(crate) struct_fields: Option<Vec<&'a Field>>,
```

Comment:
As commented already, here it's sufficient to use `pub`.

Reply:
5b74eff